backend/api/groups.js

```diff
@@ -11,27 +12,26 @@ router.get("/", (req, res) => {
 });
 
 router.post("/", (req, res) => {
-  const { name, description, avatarURL } = req.body;
+  const { name, description, avatarURL, user_id } = req.body;
+  let group_id = uuidv4();
   const newGroup = new Group({
-    group_id: uuidv4(),
+    group_id: group_id,
     name: name,
     description: description,
     avatarURL: avatarURL,
   });
   newGroup
     .save()
-    .then(() =>
-      res.json({
-        group: newGroup,
-        message: "Added group successfully",
-      })
-    )
+    .then(() => res.json(newGroup))
```

Comment:
Can this use the then((savedGroup) => res.json(savedGroup)) pattern?
Probably not the biggest deal but this doesn't exclude the database-assigned "_id" field from front-end state, and ensures front-end state is fully aligned with the database (I think?)

Reply:
Yes I'll update that